frontend/src/components/main-nav.js

```diff
@@ -42,13 +43,18 @@ class MainNav extends React.Component {
           strokeWidth={1}
         />
         <span className="caption">Log Out</span>
-      </a>
+      </NavLink>
     );
   }
 
   getLink(link, linkName, iconName) {
     return (
-      <a href={`/${link}`} className="icon-caption">
+      <NavLink
```

Comment:
changing from the a tag to NavLink will cause a bug because it won't refresh the page so the LeftNav doesn't reset (on the advisor portal click on a client and go to plan, then click on LeftNav and then go to configuration. LeftNav won't reset here)